eng/scripts/CodeCheck.ps1

```diff
@@ -0,0 +1,119 @@
+#requires -version 5
+<#
+.SYNOPSIS
+This script runs a quick check for common errors, such as checking that Visual Studio solutions are up to date or that generated code has been committed to source.
+#>
+param(
+    [switch]$ci
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 1
+Import-Module -Scope Local -Force "$PSScriptRoot/common.psm1"
```

Comment:
Where is anything from this module used?

Reply:
it's not anymore. it was in earlier drafts of the PR. I'll remove this.